game.py

```diff
@@ -20,7 +20,29 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+    wrong_guesses_list = []
+    wrong_guesses_count = 0
+    continue_playing = True
+
+    while continue_playing:
+        letter = get_letter_from_user(correct_letter_guess_statuses, wrong_guesses_list)
+        if letter in snowman_word:
+            correct_letter_guess_statuses[letter] = True
+        else:
+            wrong_guesses_list.append(letter)
+            wrong_guesses_count += 1
+
+        print_snowman_graphic(wrong_guesses_count)
+        print_word_progress_string(snowman_word, correct_letter_guess_statuses)
+        print(f"You have {wrong_guesses_count} wrong guesses: {wrong_guesses_list}")
```

Comment:
As I was running your code, I did notice that everything prints to the console without much space between which decreases readability. In the future, when we are working with command line outputs, feel free to play around with line breaks to make the spacing easier to see!